packages/config-plugins/src/utils/Updates.ts

```diff
@@ -125,3 +127,67 @@ export function getRuntimeVersion(
     }" is not a valid runtime version. getRuntimeVersion only supports a string, "sdkVersion", or "nativeVersion" policy.`
   );
 }
+
+export function getSDKVersion(config: Pick<ExpoConfigUpdates, 'sdkVersion'>): string | null {
+  return typeof config.sdkVersion === 'string' ? config.sdkVersion : null;
+}
+
+export function getUpdatesEnabled(config: Pick<ExpoConfigUpdates, 'updates'>): boolean {
+  return config.updates?.enabled !== false;
+}
+
+export function getUpdatesTimeout(config: Pick<ExpoConfigUpdates, 'updates'>): number {
+  return config.updates?.fallbackToCacheTimeout ?? 0;
+}
+
+export function getUpdatesCheckOnLaunch(
+  config: Pick<ExpoConfigUpdates, 'updates'>,
+  expoUpdatesPackageVersion?: string | null
+): 'NEVER' | 'ERROR_RECOVERY_ONLY' | 'ALWAYS' {
+  if (config.updates?.checkAutomatically === 'ON_ERROR_RECOVERY') {
+    // native 'ERROR_RECOVERY_ONLY' option was only introduced in 0.11.x
+    if (expoUpdatesPackageVersion && semver.gte(expoUpdatesPackageVersion, '0.11.0')) {
+      return 'ERROR_RECOVERY_ONLY';
+    }
+    return 'NEVER';
+  } else if (config.updates?.checkAutomatically === 'ON_LOAD') {
+    return 'ALWAYS';
+  }
+  return 'ALWAYS';
+}
+
+export function getUpdatesCodeSigningCertificate(
+  projectRoot: string,
+  config: Pick<ExpoConfigUpdates, 'updates'>
+): string | undefined {
+  const codeSigningCertificatePath = config.updates?.codeSigningCertificate;
+  if (!codeSigningCertificatePath) {
+    return undefined;
+  }
+
+  const finalPath = path.join(projectRoot, codeSigningCertificatePath);
+  if (!finalPath || !fs.existsSync(finalPath)) {
+    throw new Error(`File not found at \`updates.codeSigningCertificate\` path: ${finalPath}`);
+  }
+
+  // need to format it as a single line string with `\n` characters actually in the string
+  return fs.readFileSync(finalPath, 'utf8').replace(/\n/g, '\\n');
```

Comment:
This should be the job of the code that is storing/transmitting the certificate, since it knows the escaping/encoding rules of the storage/transmission format.